### DISCUSSION TITLE
Change keybindings

### DISCUSSION THREAD

**Discussion by ghost** [artempyanykh/marksman#195] [Q&A] [answered]
I would like to know if it is possible to change the keybindings.

The reason is that in Helix, for example, hover preview  does not work when pressing `K`. 

And are there any docs? I mean, instead of the demo gifs I would like to see a text documentation.

**Comment by artempyanykh** [comment, marked as answer]
Hi @bambirombi! This is really about the editor you use rather than Marksman. Marksman just provides service to an editor in the form of request/response. For Helix you probably want to look at https://docs.helix-editor.com/keymap.html#space-mode and https://docs.helix-editor.com/remapping.html

**Reply by ghost** [reply]
If someone see this, in helix you have to do `SPACE + k`
